.env.template

```diff
@@ -47,7 +48,7 @@ GA_CLIENT_EMAIL=
 
 # Hyrax application setting
 DERIVATIVES_PATH=/shared/derivatives/
-FITS_PATH=/fits/fits-1.0.5/fits.sh
+FITS_PATH=/fits/fits-1.3.0/fits.sh
```

Comment:
shall we remove the version from the fits path, so we can install any version we like without this faff?

Reply:
I think it makes sense to remove the versioning from the path - this also causes problems with the separate data import script which invokes fits. Perhaps a symlink approach?

Reply:
agreed, I'll have a look at that